scripts/create_test_data.py

```diff
@@ -99,13 +125,14 @@ def create_mill():
     for i in range(0, size):
         sgs = swagger_client.SimpleGsmShade()
         sgs.shade = random.choice(shade)
-        sgs.min_gsm = randint(0, 500)
-        sgs.max_gsm = randint(0, 500)
+        sgs.min_gsm = randint(30, 50)
+        sgs.max_gsm = randint(51, 500)
         sgs.mill = mill
         print "Creating SimpleGsmShade: %s(%s-%s) for Mill: %s" % (sgs.shade, sgs.min_gsm, sgs.max_gsm, mill.name)
-        all_sgs.append(sgs_api.create_simple_gsm_shade_using_post(sgs))
-    return mill_api.get_mill_using_get(mill.id)
-
+        sgs_api.create_simple_gsm_shade_using_post(sgs)
+    return mill
```

Comment:
Sorry. I was trying to do create for derived gsm in the same function. Missed to place the code back